packages-next/auth/src/index.ts

```diff
@@ -339,6 +334,17 @@ export function createAuth<GeneratedListTypes extends BaseGeneratedListTypes>({
           return (await adminPageMiddleware(args)) ?? keystoneConfig?.ui?.pageMiddleware?.(args);
         },
         enableSessionItem: true,
+        isAccessAllowed: async (context: KeystoneContext) => {
+          // Allow access to the adminMeta data from the /init path to correctly render that page
+          // even if the user isn't logged in (which should always be the case if they're seeing /init)
+          const headers = context.req?.headers;
+          const url = headers?.referer ? new URL(headers.referer) : undefined;
+          return (
+            (url?.pathname === '/init' && url?.host === headers?.host) ||
```

Comment:
This should check that there are no items in the list otherwise people can just fake this and get the admin meta.